### PR TITLE
Make pagination align with application cards

### PR DESCRIPTION
**Pull request by willmcb**
## Context

Align pagination with application cards so that app matches the design.

## Changes proposed in this pull request

**Before:**
Screenshot 2020-06-12 at 09 29 38

**After:**
Screenshot 2020-06-12 at 09 27 45

## Guidance to review

- is this correct from a design perspective. Note that if there are a lot of applications then the pagination will fold up into two lines:

Screenshot 2020-06-12 at 09 28 43

## Link to Trello card

https://trello.com/c/FKkvBXXs/2117-title-pagination-should-be-aligned-with-the-list-within-the-right-hand-column

## Things to check

- [x] This code doesn't rely on migrations in the same Pull Request
- [x] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [x] API release notes have been updated if necessary
- [x] New environment variables have been [added to the Azure config](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training#azure-hosting-devops-pipeline)

### CODE REVIEW

File: app/frontend/styles/_filter.scss [hunk 1]
```diff
@@ -30,6 +30,10 @@
   box-shadow: none;
 }
 
+.moj-pagination__list {
+  padding-left: 3px;
```
Comment (adamsilver) [inline app/frontend/styles/_filter.scss:34]: Why is this needed? (It's not on the prototype.)

File: app/frontend/styles/_filter.scss [hunk 2]
```diff
@@ -30,6 +30,10 @@
...
+}
+
 .app-filter {
   display: none;
```
Comment (adamsilver) [inline app/frontend/styles/_filter.scss:38]: I know that this isn't part of the review but this means without JS the user won't be able to see filters on small screens which is inaccessible. See prototype code.

File: app/frontend/styles/_filter.scss (hunk 1 above)
Comment (willmcb) [reply app/frontend/styles/_filter.scss:34]: For some reason without this it is the chevrons don't fit correctly.

**Comment by willmcb** [comment]
> Works nicely for me locally. I wondered why the padding was needed also. Removing didn't seem to break the layout for me.

Odd it looks like this for me: 
Screenshot 2020-06-12 at 11 45 01

**Comment by adamsilver** [comment]
The padding is unfortunately needed for now because when you're on page 2 the arrow is partially cut off by the filter layout divs which have overflow hidden.

Ideally the layout needs to change to avoid the overflow hidden technique.

For now I’d say merge with the padding
